packages/dashboard/src/components/elements/button/button.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Button as CButton } from '@deriv/components';
+
+const Button: React.FC<TButtonProps> = ({ className, children, onClick, ...props }) => {
+    return (
+        <CButton className={classNames('dw-btn', className)} onClick={onClick} {...props}>
+            {children}
+        </CButton>
+    );
+};
+
+type TButtonProps = {
+    className?: string;
+    large?: boolean;
+    primary?: boolean;
+    tertiary?: boolean;
+    children: React.ComponentType | React.ElementType | React.ReactElement;
+    onClick?: React.MouseEventHandler<HTMLButtonElement>;
+};
```

Comment:
Can we change `type` to `interface`, since interfaces provide more capabilities it's best to practice them in our code base.

Reply:
with the latest typescript funcitonality, the usage and capabilities of type and interface become very similiar, i agree that we need to use interface for props, types can be used for function, since type has capabilities to also typecheck the return of the function and support intersection and union